cmd/apichecker/README.md

```diff
@@ -0,0 +1,82 @@
+# GoCryptoTrader package Apichecker
+
+<img src="https://github.com/thrasher-corp/gocryptotrader/blob/master/web/src/assets/page-logo.png?raw=true" width="350px" height="350px" hspace="70">
+
+
+[![Build Status](https://travis-ci.org/thrasher-corp/gocryptotrader.svg?branch=master)](https://travis-ci.org/thrasher-corp/gocryptotrader)
+[![Software License](https://img.shields.io/badge/License-MIT-orange.svg?style=flat-square)](https://github.com/thrasher-corp/gocryptotrader/blob/master/LICENSE)
+[![GoDoc](https://godoc.org/github.com/thrasher-corp/gocryptotrader?status.svg)](https://godoc.org/github.com/thrasher-corp/gocryptotrader/cmd/apichecker)
+[![Coverage Status](http://codecov.io/github/thrasher-corp/gocryptotrader/coverage.svg?branch=master)](http://codecov.io/github/thrasher-corp/gocryptotrader?branch=master)
+[![Go Report Card](https://goreportcard.com/badge/github.com/thrasher-corp/gocryptotrader)](https://goreportcard.com/report/github.com/thrasher-corp/gocryptotrader)
+
+
+This apichecker package is part of the GoCryptoTrader codebase.
+
+## This is still in active development
+
+You can track ideas, planned features and what's in progresss on this Trello board: [https://trello.com/b/ZAhMhpOy/gocryptotrader](https://trello.com/b/ZAhMhpOy/gocryptotrader).
+
+Join our slack to discuss all things related to GoCryptoTrader! [GoCryptoTrader Slack](https://join.slack.com/t/gocryptotrader/shared_invite/enQtNTQ5NDAxMjA2Mjc5LTc5ZDE1ZTNiOGM3ZGMyMmY1NTAxYWZhODE0MWM5N2JlZDk1NDU0YTViYzk4NTk3OTRiMDQzNGQ1YTc4YmRlMTk)
+
+## Current Features for apichecker
+
++ Checks for API updates
++ Can automatically update Trello checklist for the updates required
+
+## Usage
+
++ To run a real check for updates, parse Trello API info as flags or add them to the updates.json file and use the following command from apichecker folder in GCT:
+```bash
+go build && apichecker.exe --verbose
+```
+
++ Upon addition of a new exchange, to update Trello checklist and to add the exchange to updates.json the following would need to be done:
+
+#### HTML Scraping method:
+```go
+func TestAdd(t *testing.T) {
+	t.Parallel()
+	data := HTMLScrapingData{TokenData: "div",
+		Key:    "class",
+		Val:    "col-md-12",
+		RegExp: "col-md-12([\\s\\S]*?)clearfix",
+		Path:   "https://localbitcoins.com/api-docs/"}
+	err := Add("LocalBitcoins", htmlScrape, data.Path, data, true, &testConfigData)
+	if err != nil {
+		t.Error(err)
+    }
+}
+```
+
+#### Github SHA Check Method:
+```go
+func TestAdd(t *testing.T) {
+	t.Parallel()
+	data := GithubData{Repo: "LBank-exchange/lbank-official-api-docs"}
+	err := Add("Lbank", github, fmt.Sprintf(githubPath, data.Repo), data, false, &configData)
+	if err != nil {
+		t.Error(err)
+    }
+}
+```
+
```

Comment:
It would also be nice to mention that this tool supports Trello integration for those who are interested and examples of how to use it